chain/network-primitives/src/network_protocol/edge.rs

```diff
@@ -3,6 +3,15 @@ use near_crypto::{KeyType, SecretKey, Signature};
 use near_primitives::borsh::maybestd::sync::Arc;
 use near_primitives::hash::CryptoHash;
 use near_primitives::network::PeerId;
+use once_cell::sync::Lazy;
+
+use crate::time;
+
+// We'd treat all nonces that are below this values as 'old style' (without any expiration time).
+// And all nonces above this value as new style (that would expire after some time).
+// This value is set to August 8, 2022.
+pub const EDGE_MIN_TIMESTAMP_NONCE: Lazy<time::Utc> =
```

Comment:
add a TODO to remove it, once support for old nonce semantics is removed.

Reply:
done.